uberfire-rest/uberfire-rest-backend/src/main/java/org/guvnor/rest/backend/JobRequestHelper.java

```diff
@@ -437,22 +426,22 @@ public JobResult deployProject(final String jobId,
     }
 
     public JobResult removeOrganizationalUnit(final String jobId,
-                                              final String organizationalUnitName) {
+                                              final String spaceName) {
```

Comment:
I'm not sure if the actual name of the method is correct. Shouldn't this method be called removeSpace?

Reply:
Maybe. I think it is better to rename it. Doing it tomorrow.